actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -249,39 +258,42 @@ def signed_or_encrypted
 
       private
 
-        def upgrade_legacy_signed_cookies?
+        def rotate_out_legacy_signed_cookies?
           request.secret_token.present? && request.secret_key_base.present?
         end
 
-        def upgrade_legacy_hmac_aes_cbc_cookies?
+        def rotate_out_legacy_aes_cbc_hmac_cookies?
```

Comment:
Let's just leave these methods to say `upgrade`. That's a different job than the rest of the code interpreting it as meaning "apply rotation".